src/main/java/hobbylist/logic/commands/StatusCommand.java

```diff
@@ -0,0 +1,85 @@
+package hobbylist.logic.commands;
```

Comment:
Is there a need to create a new command for statuses, rather than using the edit command?

Reply:
I have removed the StatusCommand and StatusCommandParser already. Activity's status can now be included using 'add' and 'edit' command only. 👍